index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Setup and Mobile First Web App</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+    <div class="wrapper">
+        
+        <header class="header">
+            
+                <a href="./index.html" class="my-logo">My Logo</a>
+                <nav class="navbar">
+                    <ul>
+                        <li class="menu-bar">                            
+                            <button type="button" class="bar"><i class="fa fa-bars"></i></button>
+                    </li>
+                </ul>
+            </nav>
+            
+        </header>
+        
+        <div class="container-homepage">     
+            <section id="#about" class="section-about">
```

Comment:
Great job @amare1990 🤝

- [ ] Just like the previous reviewer stated, your app does not fill all the spaces on larger mobile screens. Kindly find out the issue and fix it.
- Your design
![design](https://user-images.githubusercontent.com/63917725/186246054-1b794063-b206-457b-b406-e8cfde45e5ca.PNG)

Reply:
@1GORDON, can you give me hint for filling the entire width for the header? I tried much but I could not.  Both the header and the homepage container are direct children of the container parent div.